### DISCUSSION TITLE
Two Bosch Indego Adapters with the same name

### DISCUSSION THREAD

**Discussion by friedpa** [DEV2DEV-DE/ioBroker.boschindego#7] [General]
Hi Bernd, there are now two Adapters for the Bosch Indego @ IOBroker. How do I make sure to get your Adapter running?

**Comment by DEV2DEV-DE** [comment]
Hallo Paul,
der Kollege TA2k hat "netterweise" den Adapter "gekapert" und seinen ohne Abstimmung ins Beta-Repo gebracht.
Den meinigen kannst Du nur im Expertenmodus über "die Katze" installieren.
Man sollte das am Logo erkennen können. Meiner zeigt das Bild des Indego, der andere zeigt ein Icon.

Wie wir mit solchen - reichlich unfreundlichen - Aktionen künftig umgehen, werden wir im Entwickler-Meeting kommende Woche besprechen.
Das da jetzt Verwirrung entsteht, habe ich bereits befürchtet. Das tut mir leid, ist aber nun von meiner Seite nicht mehr zu ändern.
Der Kollege scheint einen etwas überhöhten Geltungsdrang zu haben.

![image](https://github.com/DEV2DEV-DE/ioBroker.boschindego/assets/95883579/f65d6d97-52f9-40f8-aaf6-e56dde894979)

**Comment by friedpa** [comment]
Hallo Bernd, 
habe Deinen Adapter installiert. Leider aus sessonalen Gründen habe ich meinen Indego aber bereits eingewintert, deswegen kann ich nur nächstes Jahr testen, außer Du brauchst etwas dringendes, dann würde ich diesen wieder rausräumen. Vielen Dank für Deine Mühen überhaupt so einen Adapter zu entwickeln.
Liebe Grüße
Paul

**Reply by DEV2DEV-DE** [reply]
Hallo Paul,
werde meinen Shaun aufgrund der Witterung demnächst auch in den Keller schicken.
Eilt nicht.

Gruß
Bernd
